benchmarks/benchmarks/solarposition_numba.py

```diff
@@ -0,0 +1,44 @@
+"""ASV benchmarks for solarposition.py using numba.
+
+We use a separate module so that we can control the pvlib import process
+using an environment variable. This will force pvlib to compile the numba
+code during setup.
+
+Try to keep relevant sections in sync with benchmarks/solarposition.py
+"""
+
+from pkg_resources import parse_version
+import pandas as pd
+
+import os
+os.environ['PVLIB_USE_NUMBA'] = '1'
```

Comment:
I think the environment var might not be needed now that you added numba to the env specs

Reply:
With setting the environment variable I see this:

```
(pvlib38ci) 8:22:16 holmgren@holmgren-mbp.local benchmarks solposasv ? asv run --bench SolarPosition --show-stderr
· Creating environments
· Discovering benchmarks
· Running 22 total benchmarks (1 commits * 2 environments * 11 benchmarks)
[  0.00%] · For pvlib-python commit 30d784e7 <master>:
[  0.00%] ·· Building for conda-py3.6-ephem3.7.6.0-numba-numpy1.12.0-pandas0.22.0-pytables3.6.1-scipy1.2.0
[  0.00%] ·· Benchmarking conda-py3.6-ephem3.7.6.0-numba-numpy1.12.0-pandas0.22.0-pytables3.6.1-scipy1.2.0
[  2.27%] ··· Running (solarposition.SolarPosition.time_calc_time--)...........
[ 27.27%] ··· solarposition.SolarPosition.time_calc_time                                                       363±3μs
[ 29.55%] ··· solarposition.SolarPosition.time_ephemeris                                                    22.6±0.5ms
[ 31.82%] ··· solarposition.SolarPosition.time_ephemeris_localized                                          22.4±0.2ms
[ 34.09%] ··· solarposition.SolarPosition.time_nrel_earthsun_distance                                         26.2±1ms
[ 36.36%] ··· solarposition.SolarPosition.time_pyephem                                                         416±2ms
[ 38.64%] ··· solarposition.SolarPosition.time_spa_python                                                      134±4ms
[ 40.91%] ··· solarposition.SolarPosition.time_sun_rise_set_transit_ephem                                   53.5±0.6ms
[ 43.18%] ··· solarposition.SolarPosition.time_sun_rise_set_transit_geometric                              4.38±0.03ms
[ 45.45%] ··· solarposition.SolarPosition.time_sun_rise_set_transit_spa                                     22.1±0.3ms
[ 47.73%] ··· solarposition_numba.SolarPositionNumba.time_spa_python                                          28.1±1ms
[ 50.00%] ··· solarposition_numba.SolarPositionNumba.time_sun_rise_set_transit_spa                          6.90±0.1ms
```

Without setting the environment variable (comment out the line), I see this:

```
(pvlib38ci) 8:20:05 holmgren@holmgren-mbp.local benchmarks solposasv ? asv run --bench SolarPosition --show-stderr
· Creating environments
· Discovering benchmarks
· Running 22 total benchmarks (1 commits * 2 environments * 11 benchmarks)
[  0.00%] · For pvlib-python commit 30d784e7 <master>:
[  0.00%] ·· Building for conda-py3.6-ephem3.7.6.0-numba-numpy1.12.0-pandas0.22.0-pytables3.6.1-scipy1.2.0
[  0.00%] ·· Benchmarking conda-py3.6-ephem3.7.6.0-numba-numpy1.12.0-pandas0.22.0-pytables3.6.1-scipy1.2.0
[  2.27%] ··· Running (solarposition.SolarPosition.time_calc_time--)...........
[ 27.27%] ··· solarposition.SolarPosition.time_calc_time                                                      378±20μs
[ 29.55%] ··· solarposition.SolarPosition.time_ephemeris                                                      23.8±1ms
[ 31.82%] ··· solarposition.SolarPosition.time_ephemeris_localized                                          22.7±0.5ms
[ 34.09%] ··· solarposition.SolarPosition.time_nrel_earthsun_distance                                         38.2±7ms
[ 36.36%] ··· solarposition.SolarPosition.time_pyephem                                                        447±10ms
[ 38.64%] ··· solarposition.SolarPosition.time_spa_python                                                      138±5ms
[ 40.91%] ··· solarposition.SolarPosition.time_sun_rise_set_transit_ephem                                   53.1±0.5ms
[ 43.18%] ··· solarposition.SolarPosition.time_sun_rise_set_transit_geometric                               4.56±0.2ms
[ 45.45%] ··· solarposition.SolarPosition.time_sun_rise_set_transit_spa                                     22.3±0.7ms
[ 47.73%] ··· solarposition_numba.SolarPositionNumba.time_spa_python                                          29.3±2ms
[ 47.73%] ···· /Users/holmgren/git_repos/pvlib-python/benchmarks/env/4b3141485b42c1a3d28c30cd810d4559/lib/python3.6/site-packages/pvlib/solarposition.py:266: UserWarning: Reloading spa to use numba
               warnings.warn('Reloading spa to use numba')

[ 50.00%] ··· solarposition_numba.SolarPositionNumba.time_sun_rise_set_transit_spa                          6.81±0.2ms
[ 50.00%] ···· /Users/holmgren/git_repos/pvlib-python/benchmarks/env/4b3141485b42c1a3d28c30cd810d4559/lib/python3.6/site-packages/pvlib/solarposition.py:266: UserWarning: Reloading spa to use numba
               warnings.warn('Reloading spa to use numba')
```